### ISSUE TITLE
Add ability to take screenshots

### ISSUE THREAD

**Issue by AnugrahJames** [MarcAnt01/Fluent-Screen-Recorder#35]
I thought it would be nice to be able to take a screenshot while recording. It would be a handy feature.

**Comment by MarcAnt01** [comment]
Hi, this app is thought as a screen recorder, I often use Snip&Sketch app for screenshots. Can you tell me why you would like this feature implemented in Fluent Screen recorder?

**Comment by AnugrahJames** [comment]
Yes, I get you. Snip&Sketch is fine for screenshots. But what I meant was to capture a pic from the recording window while recording is in progress. Example: In some phones you can take a photo while recording a video and they are saved separately.

**Comment by MarcAnt01** [comment]
Ah, got now what you mean, yeah, in this case it could definitely make sense😁 👍

**Comment by Jaiganeshkumaran** [comment]
@AnugrahJames You can record the video and then extract the frame you want using Photos app

**Comment by AnugrahJames** [comment]
Sorry for closing the issue. It was by mistake.

**Comment by AnugrahJames** [comment]
@Jaiganeshkumaran Yes. But being able to that in the app makes it more independent.
